01-node-tutorial/answers/writeWithPromisesThen.js

```diff
@@ -0,0 +1,18 @@
+const {writeFile, readFile} = require("fs").promises;
+writeFile('temp.txt', 'Checkout my line 1')
```

Comment:
Your code works great! If you want all lines to show when the code finishes, you'll also need to use the [appendFile](https://nodejs.org/api/fs.html#filehandleappendfiledata-options) function. That way you can overwrite the content of the file when writing the first line and append when writing the other 2 lines.